dockerfiles/cfme_tests_base/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# Our custom docker image with app
```

Comment:
if we set up the docker images to expect mounted volumes, we can make them more public